themes/digital.gov/src/scss/new/shortcodes/_ring.scss

```diff
@@ -23,6 +23,7 @@
 
   h2 {
     @include u-font("sans", "lg");
+    font-size: 1.75rem;
```

Comment:
I'm fine with adding another `8px` to be consistent  with other `h2`'s for this component.

Let's use the [size](https://designsystem.digital.gov/design-tokens/typesetting/font-size/#using-size-tokens) function:

```css
    font-size: size("sans", 11);
```


Let's also update `$font-size` to `units(4)` on line 13 to improve the border and header alignment.

<img width="482" alt="Screenshot 2024-06-21 at 11 11 23 AM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/7683a525-b641-4772-a39a-cae0bfd4d3c9">

Reply:
Made the changes! Thank you @nick-mon1